app/views/events/team.html.erb

```diff
@@ -32,6 +32,24 @@
   <% end %>
 </h1>
 
+<% if @event.parent.present? %>
+  <%= render "callout", title: "The team behind #{@event.parent.name} also has access to #{@event.name}", type: "info", icon: "leader" do %>
+    <p>Because <%= @event.name %> is a subsidiary of <%= @event.parent.name %>, all team members of <%= @event.parent.name %> can access and view this organization.</p>
```

Comment:
```suggestion
    <p>Because <%= @event.name %> is a sub-organization of <%= @event.parent.name %>, all team members of <%= @event.parent.name %> can access and view this organization.</p>
```